### PR TITLE
DOCS-5533 Exchange information in ACM

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-5533 Exchange information in ACM

From Fernando's Google doc: https://docs.google.com/document/d/1dQ3UPct7xFDSOEsSvWjryt8VJJkCEwEmguYNkdrqgUc/edit

~~Fernando:~~
~~* The doc says API Console info "seems to be extracted from exchange but we get it from amf". In this case, is AMF API Modeling Framework and not AML Modeling Framework?~~
~~* Does the doc have a complete list of each data field? The doc listed only asset names for the API Carousel component, but Jesus used that as an example and listed API name, description, and icon.~~

~~Thank you!~~

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Fernando explained in the Google doc that AMF is AML Modeling Framework and AML is Anything Modeling Language. Updated the PR with this info.

**Review by ferferrarazzo — COMMENTED** [review]
LGTM

**Comment by wwarner-salesforce** [comment]
Fernando notes that the queries to the AMF models are made to the AMF Service. That is, AMF Service is the API that exposes the AMF models.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -70,6 +76,8 @@ Endpoints are displayed in alphabetical order.
 
 Console components include API Console, API Console Documentation, API Console Instances, API Console Navigation, and API Console Request Panel. In Community Builder, you can add multiple console components to the same page. Changes to a console component are visible in all other console components on that page that display the same API. For example, selecting the *Summary* tab in an API Console results in *Summary* tab selection in all API Consoles on the same page that display the same API.
 
+Data shown from Anypoint Platform: API instances and documentation from the AML Modeling Framework (AMF). AML is the Anything Modeling Language.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: THis should say the Anypoint component known by the user/admin where the data comes from. Where do this info comes from, ultimately from the visible product set? is it from API Manager? Exchange? these are the component names we should use, as other things like AML or AMF are not visibile or understandable for the users.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 2]
```diff
@@ -96,7 +104,9 @@ Method Label Head Color:: Specifies the HEAD method label color.
 
 == API Console Documentation
 
-The API Console Documentation component shows the documentation from Anypoint Exchange about an API.
+The API Console Documentation component shows the documentation about an API.
+
+Data shown from Anypoint Platform: API documentation from Anypoint Exchange.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: add link to exchange docs homepage

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 3]
```diff
@@ -120,7 +130,9 @@ Headline Font Family (Optional):: Specifies the API console headline font family
 
 == API Console Instances
 
-The API Console Instances component shows the list of instances of an API.
+The API Console Instances component shows the list of instances of an API, also known as endpoints.
+
+Data shown from Anypoint Platform: API instances from Anypoint API Manager.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: add link to APIManager docs homepage

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 4]
```diff
@@ -130,6 +142,8 @@ API:: Specifies the API whose instances are displayed.
 
 The API Console Navigation component shows the API console component's navigation.
 
+Data shown from Anypoint Platform: API instances and documentation from the AML Modeling Framework (AMF). AML is the Anything Modeling Language.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: see comment above regarding AML and product names

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 5]
```diff
@@ -153,6 +167,8 @@ Method Label Head Color:: Specifies the HEAD method label color.
 
 The API Console Request Panel component shows a panel where users can test an API by making requests to it.
 
+Data shown from Anypoint Platform: API instances from Anypoint API Manager.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: add link to API Manager docs homepage

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 6]
```diff
@@ -193,6 +209,8 @@ API Version:: Adds the API version to the button label. Values are *Append*, *Pr
 
 The API Documentation Viewer component shows an API's documentation. The documentation can have multiple pages, and each page can have multiple sections.
 
+Data shown from Anypoint Platform: API documentation from Anypoint Exchange.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Add link to Exchange docs homepage

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 7]
```diff
@@ -203,6 +221,8 @@ Selected Page (Optional):: Selects the page to open.
 
 The API Header component displays the API header, including icon, name, and description.
 
+Data shown from Anypoint Platform: API icon, name, and description from Anypoint Exchange.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Add link to Exchange docs homepage

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 8]
```diff
@@ -211,6 +231,8 @@ API:: Specifies the API to show.
 
 The Application Analytics Graph component displays statistics about the use of an application, such as traffic, response time, and application health.
 
+Data shown from Anypoint Platform: Application statistics from Anypoint API Manager.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Add link to API Manager docs homepage
Double check if this comes from API Manager or Monitoring center

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 9]
```diff
@@ -219,6 +241,8 @@ Application:: Specifies the application to show.
 
 The Application Details component displays details about an application.
 
+Data shown from Anypoint Platform: Application description, application URL, redirect URIs, client ID, and client secret from Anypoint Exchange.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Add link to Exchange docs homepage

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 10]
```diff
@@ -227,6 +251,8 @@ Application:: Specifies the API to show.
 
 The Application Listing component shows applications that use an API.
 
+Data shown from Anypoint Platform: Application name, client ID, and client secret from Anypoint Exchange.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: This shouldn't be API Manager?

**Review by jdeoliveira — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 7 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 8 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Link added. Still need to double check if this comes from API Manager or Monitoring center.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 9 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Done.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (jdeoliveira) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Confirmed with eng: information in the Console comes from Anypoint Exchange

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 4 above)
Comment (jdeoliveira) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Confirmed with eng: information in the Console comes from Anypoint Exchange

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 10 above)
Comment (jdeoliveira) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Confirmed with eng: information in My Applications list comes from API Manager

**Review by jdeoliveira — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 10 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 8 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Jesus confirmed today this is API Manager.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 11]
```diff
@@ -203,6 +221,8 @@ Selected Page (Optional):: Selects the page to open.
 
 The API Header component displays the API header, including icon, name, and description.
 
+Data shown from Anypoint Platform: API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Reading this again made me realize that the Icon, name and description come from the API publication config in ACM (done in the ACM control panel when publishing an API). I suggest we remove the "data shown from Anypoint Platform line" and say something like this instead:

"The API icon, name and description shown in this component are the ones defined when the API version was added to the community in ACM control panel" (and link to the section where API publishing/editing is described.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 12]
```diff
@@ -44,6 +48,8 @@ Learn More Button Label:: Enables you to set the Learn More button label. Defaul
 
 The API Carousel component shows a horizontally scrollable set of API cards.
 
+Data shown from Anypoint Platform: API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
+
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc:52]: Reading this again made me realize that the Icon, name and description come from the API publication config in ACM (done in the ACM control panel when publishing an API). I suggest we remove the "data shown from Anypoint Platform line" and say something like this instead:

"The API icon, name and description shown in this component are the ones defined when the API version was added to the community in ACM control panel" (and link to the section where API publishing/editing is described.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 13]
```diff
@@ -27,6 +29,8 @@ Show Modal Title Line:: Shows the title line in the access request modal.
 
 The API Card component shows information about an API, including icon, name, and description.
 
+Data shown from Anypoint Platform: API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
+
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc:33]: Reading this again made me realize that the Icon, name and description come from the API publication config in ACM (done in the ACM control panel when publishing an API). I suggest we remove the "data shown from Anypoint Platform line" and say something like this instead:

"The API icon, name and description shown in this component are the ones defined when the API version was added to the community in ACM control panel" (and link to the section where API publishing/editing is described.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 14]
```diff
@@ -219,6 +241,8 @@ Application:: Specifies the application to show.
 
 The Application Details component displays details about an application.
 
+Data shown from Anypoint Platform: Application description, application URL, redirect URIs, client ID, and client secret from xref:exchange::index.adoc[Anypoint Exchange].
```
Comment (jdeoliveira) [inline modules/ROOT/pages/acm-lightning-components.adoc]: This should be API Manager

**Review by jdeoliveira — DISMISSED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 15]
```diff
@@ -10,6 +10,8 @@ These Lightning components include both the Lightning components available in al
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:10]: GREEN TEXT ONLY

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 16]
```diff
@@ -27,6 +29,8 @@ Show Modal Title Line:: Shows the title line in the access request modal.
 
 The API Card component shows information about an API, including icon, name, and description.
 
+Data shown from Anypoint Platform: API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: For parallelism with prev. sentence...:
```suggestion
Anypoint Platform shows the API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
```

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 17]
```diff
@@ -44,6 +48,8 @@ Learn More Button Label:: Enables you to set the Learn More button label. Defaul
 
 The API Carousel component shows a horizontally scrollable set of API cards.
 
+Data shown from Anypoint Platform: API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 18]
```diff
@@ -70,6 +76,8 @@ Endpoints are displayed in alphabetical order.
 
 Console components include API Console, API Console Documentation, API Console Instances, API Console Navigation, and API Console Request Panel. In Community Builder, you can add multiple console components to the same page. Changes to a console component are visible in all other console components on that page that display the same API. For example, selecting the *Summary* tab in an API Console results in *Summary* tab selection in all API Consoles on the same page that display the same API.
 
+Data shown from Anypoint Platform: API instances and documentation from xref:exchange::index.adoc[Anypoint Exchange].
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 19]
```diff
@@ -96,7 +104,9 @@ Method Label Head Color:: Specifies the HEAD method label color.
 
 == API Console Documentation
 
-The API Console Documentation component shows the documentation from Anypoint Exchange about an API.
+The API Console Documentation component shows the documentation about an API.
+
+Data shown from Anypoint Platform: API documentation from xref:exchange::index.adoc[Anypoint Exchange].
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 20]
```diff
@@ -120,7 +130,9 @@ Headline Font Family (Optional):: Specifies the API console headline font family
 
 == API Console Instances
 
-The API Console Instances component shows the list of instances of an API.
+The API Console Instances component shows the list of instances of an API, also known as endpoints.
+
+Data shown from Anypoint Platform: API instances from xref:api-manager::index.adoc[Anypoint API Manager].
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc]: GLOBAL: I will stop marking this particular suggestion. Please apply throughout.

**Review by kbostrom — APPROVED** [review]
Edit of green text only.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 11 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: I agree, that's better. Updated.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 12 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:52]: Updated.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 13 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:33]: Updated.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 14 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 16 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: You're right, that's better. Updated throughout.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 17 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated throughout.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 18 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated throughout.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 19 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated throughout.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 20 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated throughout.

**Pull request merged**